src/System.Net.Primitives/src/System/Net/IPAddressParser.cs

```diff
@@ -36,27 +35,24 @@ internal static IPAddress Parse(string ipString, bool tryParse)
                 // parse method to unpick it all. Note: we don't support
                 // port specification at the end of address and so can
                 // make this decision.
-                uint scope = 0;
-                ushort port = 0;
+                uint scope;
                 byte[] bytes = new byte[IPv6AddressBytes];
+                error = IPAddressPal.Ipv6StringToAddress(ipString, bytes, out scope);
 
-                error = Interop.NtDll.RtlIpv6StringToAddressExW(ipString, bytes, out scope, out port);
-
-                if (error == Interop.StatusOptions.STATUS_SUCCESS)
+                if (error == IPAddressPal.SuccessErrorCode)
                 {
-                    // AppCompat: .Net 4.5 ignores a correct port if the address was specified in brackes.
+                    // AppCompat: .Net 4.5 ignores a correct port if the address was specified in brackets.
```

Comment:
Nit: brackets or braces?  I don't know which is correct, but both are one character away from the original typo :smile: